### PR TITLE
Grammar/typo fixes in Readme

### PR DESCRIPTION
**Pull request by Jimmydalecleveland**
Issue: Quite a few grammatical errors and typos that made the readme harder to parse

## What I did
Fixed all the issues I saw

### CODE REVIEW

File: addons/contexts/README.md [hunk 1]
```diff
@@ -220,16 +220,16 @@ be shown at first in the toolbar menu in your Storybook.
...
-3. Additional `params` can be "appended" into an existed setups at the story level too (make sure it goes with the
+3. Additional `params` can be "appended" into an existing setup at the story level too (make sure it goes with the
    correct `title`); however, they are never be able to overridden the default setups. So it is important to have
-   non-collided names.
+   non-colliding names.
 4. The addon will persist the selected params (the addon state) between stories at run-time (similar to other
-   addons). If the active param were gone after story switching, it fallback to the default then the first. As a
-   rule of thumbs, whenever collisions made possible, always the first wins.
-5. Query parameters are supported for pre-selecting contexts param, which comes handy for visual regression testing.
+   addons). If the active params were gone after story switching, it falls back to the default then the first. As a
```
Comment (wuweiweiwu) [inline addons/contexts/README.md]: nit: fallbacks

File: addons/contexts/README.md [hunk 2]
```diff
@@ -178,13 +178,13 @@ An array of params contains a set of predefined `props` for `components`. This o
 
 (required)
 
-An unique name for representing the props.
+A unique name for representing the props.
```
Comment (wuweiweiwu) [inline addons/contexts/README.md]: not a grammar pro but shouldnt this stay `An` as unique starts with a vowel?

Comment (Jimmydalecleveland) [reply addons/contexts/README.md]: It is a vowel, but the sound is what matters. A consonant sound, which unique makes here ("yun" with a hard y).

**Review by shilman — APPROVED** [review]
Great updates. Thanks!! 🙏

**Review by shilman — APPROVED** [review]
Great updates! Thanks!! 🙏

**Pull request merged**